modules/pipelines-web-console.adoc

```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
 //
-// * layered-products-web-console.adc
+// * products-web-console.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="pipelines-web-console_{context}"]
 = {pipelines-title} in the web console
 
-{pipelines-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. You can use the OperatorHub in the {product-title} web console to install the {pipelines-title} Operator to create and modify pipeline objects.
\ No newline at end of file
+{pipelines-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. Install the {pipelines-title} Operator using the OperatorHub in the {product-title} web console. Once the Operator is installed, you can create and modify pipeline objects on *Pipelines* page of the *Developer* perspective.
```

Comment:
Maybe there is no need to highlight the `Developer perspective`, according to the description on the [working-with-pipelines-web-console](https://access.redhat.com/documentation/en-us/red_hat_openshift_pipelines/1.13/html/creating_cicd_pipelines/working-with-pipelines-web-console) page --`You can use the Administrator or Developer perspective to create and modify Pipeline, ....`
Suggest removing ` of the *Developer* perspective`

Reply:
Updated. Thanks!